src/index.scss

```diff
@@ -57,16 +62,35 @@ body {
 	color: white;
 }
 
+p {
+	margin: 0;
+}
+
```

Comment:
Why are the margin/padding not set to 0 for *? The margin 0 could be avoided here. Might want to see if this would work

Reply:
Affects a bit; let's revisit this after deadline